run-integration-test/README.md

```diff
@@ -59,15 +59,13 @@ calling interu. For example, the `schedule` profile could be used in CI on the `
 
 The following strategies are currently available:
 
-- `weighted`.
-- `use-runner`.
```

Comment:
IMO, this was good here. It lists the exact strategies without having to parse the sentences below.